src/components/interactive-map.js

```diff
@@ -24,11 +25,11 @@ const propTypes = Object.assign({}, StaticMap.propTypes, {
   minPitch: PropTypes.number,
 
   /**
-   * `onChangeViewport` callback is fired when the user interacted with the
+   * `onViewportChange` callback is fired when the user interacted with the
```

Comment:
Since we are going to change this prop, let's give the name a second round of audit. E.g. is `onViewportChange` the best choice? I have a feeling I will still find it hard to remember exactly without looking up in the docs. How about just calling it `onChange`? Would that be confusing?